### ISSUE TITLE
console commands

### ISSUE THREAD

**Issue by bog500** [lishid/OpenInv#35]
this is not an "issue" per say but most commands should be usable from console.  openinv <player> should work from the console but stuff would be listed as text instead of showing the gui.

**Comment by ShadowRanger** [comment]
Good idea. I'll see what I can do.

**Comment by Jikoo** [comment]
What kind of display did you have in mind? There can be a lot of detail to a single item.

I'm not planning on implementing this soon, but here's my idea:
`openinv <player>` for console would display a list of materials and quantity listed by slot ID:
```
1: AIR x0
2: STICK x10
...
```
`openinv <player> <slot>` would display detailed information about the item in the particular slot.

Does that sound good? If so, would we prefer the Bukkit string representation or Minecraft's JSON string representation? If we're really feeling it, we could add another optional parameter, where Bukkit variants result in the Bukkit version and Mojang variants result in the Minecraft version. If we don't feel like the optional parameter, it'd be best to use Bukkit's version for better backwards compatibility.

**Comment by bog500** [comment]
Yes, that was pretty much how I imagined it.  As for the name, I would go with the version that would be consider "best practices" by the bukkit plugin community.

**Comment by potatowin** [comment]
I like this plugin but when i give my freinds op. but they sometimes look at my inv so is there way to block this?
